flutter/lib/src/web/web_sentry_js_binding.dart

```diff
@@ -13,7 +13,7 @@ class WebSentryJsBinding implements SentryJsBinding {
   SentryJsClient? _client;
 
   @override
-  void init(Map<String, dynamic> options) {
+  void init(Map<String, dynamic> options) async {
```

Comment:
Any reason to change this to async? Not seeing any await inside this method.

Reply:
ah thanks